### PR TITLE
Add Patient, Practitioner, and Organization examples

### LINKED ISSUE #94: Additional examples for the Commonprofiles.care profiles
We've received requests internally from our devs and QAs about additional examples that show how different use cases are supported by the profiles provided by Commonprofiles.care. Specifically questions about how the use of extensions pan out in actual resources and so on. 

If we think more actors would be interested in this I suggest we could make a small effort and create some more/better/richer example resources.

### PR DESCRIPTION
**Pull request by vadi2**
Added examples for Patients, Practitioners, and Organizations.

Updated IG to use Swedish SNOMED CT version to remove some validation errors too.

Questionnaire and ServiceRequest examples still need to be done so this is WIP.

### CODE REVIEW

**Review by peturv — APPROVED** [review]
Excellent work. One thing I'm contemplating, which I think I'll post as a new issue instead, is to split up examples into separate files. And possibly sort the profiles into folders. In this suggestion we would have a folder named Patient containing all patient related profiles and examples as separate files. Still thinking about it, so lets discuss next meeting :)

File: input/fsh/expansion.fsh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Instance: expansion
```
Comment (MangalaEkanayake) [inline input/fsh/expansion.fsh:1]: Is this Instance name explain "what is it"?

Comment (vadi2) [reply input/fsh/expansion.fsh:1]: This is not an example, rather it is a configuration resource to say that the FHIR validator should use Swedish SNOMED CT when validating: https://fshschool.org/docs/sushi/tips/#choosing-a-snomed-ct-edition

**Review by MangalaEkanayake — APPROVED** [review]

File: input/fsh/OrganizationSECommonsLite.fsh [hunk 2]
```diff
@@ -84,4 +84,76 @@ Description: """This is a light weight Swedish Organization profile.
...
+  * country = "SE"
+
+Instance: maximal-organization
+InstanceOf: OrganizationSECommonsLite
+Description: "Maximal Organization instance"
+Usage: #example
+* meta.profile = Canonical(OrganizationSECommonsLite)
+* identifier[0]
+  * use = #official
+  * type = IDENTIFIER_TYPE#PRN
```
Comment (vjernelov) [inline input/fsh/OrganizationSECommonsLite.fsh:124]: According to the profile this should be:

http://terminology.hl7.org/CodeSystem/v2-0203#PRN

File: input/fsh/OrganizationSECommonsLite.fsh [hunk 3]
```diff
@@ -84,4 +84,76 @@ Description: """This is a light weight Swedish Organization profile.
...
+  * system = "urn:oid:1.2.752.29.4.19"
+  * value = "HSA-ID-EXAMPLE-1234"
+* identifier[+]
+  * use = #official
+  * type = IDENTIFIER_TYPE#XX
```
Comment (vjernelov) [inline input/fsh/OrganizationSECommonsLite.fsh:129]: According to the profile this should be:

http://terminology.hl7.org/CodeSystem/v2-0203#XX

File: input/fsh/PractitionerSECommonsLite.fsh [hunk 4]
```diff
@@ -20,3 +20,75 @@ Description: "This is a light weight Swedish Practitioner profile.
...
+  * use = #work
+
+Instance: maximal-practitioner
+InstanceOf: PractitionerSECommonsLite
+Description: "A more thorough example of a Swedish Practitioner resource."
+Usage: #example
+* meta.profile = Canonical(PractitionerSECommonsLite)
+* identifier
+  * use = #official
+  * type = IDENTIFIER_TYPE#PRN "Provider number"
```
Comment (vjernelov) [inline input/fsh/PractitionerSECommonsLite.fsh:56]: According to the profile this should be

http://terminology.hl7.org/CodeSystem/v2-0203#PRN

File: input/fsh/PractitionerSECommonsLite.fsh [hunk 5]
```diff
@@ -20,3 +20,75 @@ Description: "This is a light weight Swedish Practitioner profile.
...
 Alias: $SEBaseHumanName = http://hl7.se/fhir/ig/base/StructureDefinition/SEBaseHumanName
+
+Instance: minimal-practitioner
+InstanceOf: PractitionerSECommonsLite
+Description: "A simple example of a Swedish Practitioner resource."
+Usage: #example
+* meta.profile = Canonical(PractitionerSECommonsLite)
+* identifier
+  * use = #official
+  * type = IDENTIFIER_TYPE#PRN
```
Comment (vjernelov) [inline input/fsh/PractitionerSECommonsLite.fsh:31]: According to the profile this should be

http://terminology.hl7.org/CodeSystem/v2-0203#PRN

File: input/fsh/PractitionerSECommonsLite.fsh (hunk 4 above)
Comment (vjernelov) [reply input/fsh/PractitionerSECommonsLite.fsh:56]: Which I know see is solved using aliases :)

File: input/fsh/PractitionerSECommonsLite.fsh (hunk 5 above)
Comment (vjernelov) [reply input/fsh/PractitionerSECommonsLite.fsh:31]: Which I know see is solved using aliases :)

File: input/fsh/OrganizationSECommonsLite.fsh (hunk 2 above)
Comment (vjernelov) [reply input/fsh/OrganizationSECommonsLite.fsh:124]: Which I know see is solved using aliases :)

File: input/fsh/OrganizationSECommonsLite.fsh (hunk 3 above)
Comment (vjernelov) [reply input/fsh/OrganizationSECommonsLite.fsh:129]: Which I know see is solved using aliases :)

**Review by vjernelov — APPROVED** [review]

**Comment by niclas86** [comment]
@vadi2 can you fix the conflict and then merge and close?

**Pull request merged**